camunda-carbon-reductor-c7/docker-compose.yaml

```diff
@@ -12,8 +12,8 @@ services:
       - DataSources__Configurations__WattTime__Password=<myWattTimePassword>
     healthcheck:
       test: curl --fail 'http://localhost:80/emissions/forecasts/current?location=westus' || exit 1
-      interval: 5s
-      retries: 5
+      interval: 30s
```

Comment:
i think we can even increase that to 1 minute. what do you think @maxbehr801?

Reply:
Yes, I think 60s is less of a magic number. ;-)